### PR TITLE
Lattice Update

### PR DESCRIPTION
**Pull request by yomiadetutu1**

### CODE REVIEW

**Review by amyblais — DISMISSED** [review]

**Review by lynn915 — CHANGES_REQUESTED** [review]
@yomiadetutu1 

Can you remove "goal tracking" from the list of what we and staff use Lattice for currently? I'm not sure when (if) that will be rolled out, so I think we should hold off on including that for now. Perhaps you can replace that with 1:1s or something? Otherwise, looks good to me!

File: operations/workplace/people/README.md [hunk 1]
```diff
@@ -120,13 +120,12 @@ Resources: How-to for Staff Members
...
-### Culture Amp
+### Lattice
 
-> * Culture Amp is our Engagement and Performance Platform.
-> * The People team uses Culture Amp to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and goal tracking.
-> * Staff use Cuture Amp for performance reviews such as 90-day reviews and 360 performance reviews, for engagement and employee experience surveys, and goal tracking.
-> * To access the system go to [Culture Amp](https://mattermost.cultureamp.com) or [Okta](https://mattermost.okta.com).
-> * Note that as of March 2021, Culture Amp will not be supporting Internet Explorer 11.
+> * Lattice is our Engagement and Performance Platform.
+> * The People team uses Lattice to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and goal tracking.
```
Comment (yomiadetutu1) [inline operations/workplace/people/README.md:126]:
```suggestion
> * The People team uses Lattice to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and goal 1:1s.
```

Comment (yomiadetutu1) [inline operations/workplace/people/README.md]:
```suggestion
> * The People team uses Lattice to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and 1:1s.
```

File: operations/workplace/people/README.md [hunk 2]
```diff
@@ -120,13 +120,12 @@ Resources: How-to for Staff Members
...
+### Lattice
 
-> * Culture Amp is our Engagement and Performance Platform.
-> * The People team uses Culture Amp to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and goal tracking.
-> * Staff use Cuture Amp for performance reviews such as 90-day reviews and 360 performance reviews, for engagement and employee experience surveys, and goal tracking.
-> * To access the system go to [Culture Amp](https://mattermost.cultureamp.com) or [Okta](https://mattermost.okta.com).
-> * Note that as of March 2021, Culture Amp will not be supporting Internet Explorer 11.
+> * Lattice is our Engagement and Performance Platform.
+> * The People team uses Lattice to gain insight across the employee experience, using employee engagement surveys, continuous feedback, 360 reviews, and 1:1s.
+> * Staff use Lattice for performance reviews such as 90-day reviews and 360 performance reviews, for engagement and employee experience surveys, and goal tracking.
```
Comment (yomiadetutu1) [inline operations/workplace/people/README.md]:
```suggestion
> * Staff use Lattice for performance reviews such as 90-day reviews and 360 performance reviews, for engagement and employee experience surveys, and 1:1s.
```

**Review by lynn915 — APPROVED** [review]
Looks good to me! Thanks, Yomi!

**Pull request merged**